EIPS/eip-1363.md

```diff
@@ -11,41 +11,41 @@ requires: 20, 165
 ---
 
 ## Simple Summary
-Defines a token interface for [ERC-20](./eip-20.md) tokens that supports executing recipient code after `transfer` or `transferFrom`, or spender code after `approve`.
+Defines a token interface for [EIP-20](./eip-20.md) tokens that supports executing recipient code after `transfer` or `transferFrom`, or spender code after `approve` in a single transaction.
 
 ## Abstract
-Standard functions a token contract and contracts working with tokens can implement to make a token Payable.
+The following are the transfer, transferFrom, approve and callback functions standardized in this EIP:
 
 `transferAndCall` and `transferFromAndCall` will call an `onTransferReceived` on a `ERC1363Receiver` contract.  
 
 `approveAndCall` will call an `onApprovalReceived` on a `ERC1363Spender` contract.
 
 ## Motivation
-There is no way to execute code after a [ERC-20](./eip-20.md) transfer or approval (i.e. making a payment), so to make an action it is required to send another transaction and pay GAS twice.
+There is no way to execute code after an [EIP-20](./eip-20.md) `transfer` or `approval` (i.e. making a payment), so to make an action it is required to send another transaction and pay GAS twice.
 
 This proposal wants to make token payments easier and working without the use of any other listener. It allows to make a callback after a transfer or approval in a single transaction.
 
-There are many proposed uses of Ethereum smart contracts that can accept [ERC-20](./eip-20.md) payments. 
+There are many proposed uses of Ethereum smart contracts that can accept [EIP-20](./eip-20.md) payments. 
 
 Examples could be 
 * to create a token payable crowdsale
 * selling services for tokens 
 * paying invoices
 * making subscriptions
 
-For these reasons it was named as **"Payable Token"**.
+For these reasons it was originally named **"Payable Token"**.
 
 Anyway you can use it for specific utilities or for any other purposes who require the execution of a callback after a transfer or approval received.
 
-This proposal has been inspired by the [ERC-721](./eip-721.md) `onERC721Received` and `ERC721TokenReceiver` behaviours. 
+This proposal has been inspired by the [EIP-721](./eip-721.md) `onERC721Received` and `ERC721TokenReceiver` behaviours. 
 
 ## Specification
-Implementing contracts **MUST** implement the [ERC-1363](./eip-1363.md) interface as well as the [ERC-20](./eip-20.md) and [ERC-165](./eip-165.md) interfaces.
+Implementing contracts **MUST** implement the [EIP-1363](./eip-1363.md) interface as well as the [EIP-20](./eip-20.md) and [EIP-165](./eip-165.md) interfaces.
 
 ```solidity
 pragma solidity ^0.8.0;
 
-interface ERC1363 /* is ERC20, ERC165 */ {
+interface ERC1363 is ERC20, ERC165 {
```

Comment:
I think you should specify whether just `0xb0202a11 == type(ERC1363).interfaceId` should be registered via ERC-165, or also `type(ERC1363).interfaceId ^ type(ERC20).interfaceId ^ type(ERC165).interfaceId`. I actually think the latter is more relevant, because both ERC20 and ERC165 are required to use ERC1363.

Reply:
I think that `type(ERC20).interfaceId` is not relevant as it is clearly expressed that 
`Implementing contracts MUST implement the EIP-1363 interface as well as the EIP-20 and EIP-165 interfaces.`